docs/public/troubleshooting.md

```diff
@@ -190,6 +190,19 @@ There are two ways to increase the limit value:
 
   Don't forget to restart Consul servers to apply made changes.
 
+## Failed to get log
+
+Problem with error log "agent.server.raft: failed to get log" allows any action:
+  
+  1. Check backups in backup daemon, likes "/listbackups", "/listbackups/ID". If status equal "Successful" and if last backup is old, then we nees to
+     data loss
+  2. If you have a time to start pods, then connect to pod and delete raft.db file on all pods. If you don't have a time, then connect to kubernetes node in directory with consul PV and delete this file.
+  3. Then you need restart pods. Consul is reload, but part of data is loss.
+  4. Trying restore last backup through backup daemon. If backup daemon don't work, then reload.
+  5. Consul must be work. Check business applications - key manager and configuration-server. If they are failure on authentication to consul, then change secrets to correct from consul-acl-bootstral-secret. Then reboot.
```

Comment:
I guess need to rephrase, remove "Consul must be work" and leave it as "Check business applications...."